versioned_docs/version-latest/intro.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Getting Started
+sidebar_position: 1
+description: A Getting Started guide for the Subspace Network
+keywords:
+    - Getting Started
+    - Learn
+    - Farming
+    - Development
+---
+
+# 👋Welcome
+The Subspace Network is an ambitious layer zero protocol which is the first scalable, secure, & decentralized infrastructure layer for the Web3 ecosystems. 
+
+## ❓ Learn About the Subspace Network 
+---
+- [What is the Subspace Network](https://subspace.network/technology)
+- [Frequently Asked Questions](https://subspace.network/faq)
+- [Whitepaper - *Summarized*](https://subspace.network/news/subspace-network-whitepaper)
+- [Whitepaper - *Full Length*](https://assets.website-files.com/61526a2af87a54e565b0ae92/617759c00edd0e3bd279aa29_Subspace_%20A%20solution%20to%20the%20farmer%27s%20dilemma.pdf)
+
+## 🤝 Participate on the Network
+---
+Our goal is to bring an extremely low barrier to entry for participating on consensus, As long as you meet the simple requirements below you can get started below.
+
+
+### - [Start Farming with Subspace CLI](../protocol/cli)
```

Comment:
This link is broken for me,` ./protocol/cli` should work.

Reply:
Resolved in commit 97dd5cf & 9a76b88